### PR TITLE
add observation_budget_consumed to progress

### PR DESCRIPTION
**Pull request by oliviakim95**

### CODE REVIEW

File: test/test_object.py [hunk 1]
```diff
@@ -108,6 +108,7 @@ def experiment(self):
       'progress': {
         'object': 'progress',
         'observation_count': 3,
+        'observation_budget_consumed': 3,
```
Comment (alexandraj777) [inline test/test_object.py]: Can we test with a non-integer float?

**Comment by oliviakim95** [comment]
blocked until `multitask-web` goes live

**Comment by alexandraj777** [comment]
Why blocked? Seems like we can get this out as soon as the API field is live?

**Comment by oliviakim95** [comment]
@alexandraj777 The API field isn't live yet - all the multitask changes are being pushed to the `multitask-web` feature branch but when we merge that into `master`, we'll be ready to merge this

**Pull request merged**